Snowflake.Data/Core/Session/SFSessionProperty.cs

```diff
@@ -101,17 +102,19 @@ class SFSessionPropertyAttr : Attribute
 
     class SFSessionProperties : Dictionary<SFSessionProperty, String>
     {
-        static private SFLogger logger = SFLoggerFactory.GetLogger<SFSessionProperties>();
+        private static SFLogger logger = SFLoggerFactory.GetLogger<SFSessionProperties>();
 
         // Connection string properties to obfuscate in the log
-        static private List<SFSessionProperty> secretProps =
+        private static List<SFSessionProperty> secretProps =
             new List<SFSessionProperty>{
                 SFSessionProperty.PASSWORD,
                 SFSessionProperty.PRIVATE_KEY,
                 SFSessionProperty.TOKEN,
                 SFSessionProperty.PRIVATE_KEY_PWD,
                 SFSessionProperty.PROXYPASSWORD,
             };
+        
+        private const string AccountRegexString = "^\\w[\\w.-]+\\w$";

```

Comment:
what about underscore ? https://docs.snowflake.com/en/sql-reference/sql/create-account#required-parameters

Reply:
underscore is already covered by \w pattern